### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by fumimowdan**
Update Readme with nginx certificates steps

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,9 +59,20 @@ Create your `.env` from the `.env.example` template
 
 ### Docker based development setup
 
+There are two ways to use this approach:
+
 1. Run `tilt up -- --local-app` to launch the app on <http://localhost:3000>
+   This will setup all the dependencies for you and run the server locally ie not in a docker image
+   It is convienient when developing because you do not have to constantly rebuilt the docker image.
+
+2. Run `tilt up` that way you will be building the image definied by the Dockerfile
```
Comment (eddieleeper) [inline README.md:68]: typo `defined`

File: README.md [hunk 2]
```diff
@@ -59,9 +59,20 @@ Create your `.env` from the `.env.example` template
...
+   This approach is a closer replication to production condition
```
Comment (eddieleeper) [inline README.md:69]: Suggest rewording to `This approach is more closely aligned with a production environment.`

**Review by eddieleeper — APPROVED** [review]
LGTM - couple of small observations.

File: README.md (hunk 2 above)
Comment (fumimowdan) [reply README.md]: :recycle:
